### PR TITLE
Update mathematical-operations.md

### PR DESCRIPTION
**Pull request by cmcaine**
Apply some remaining review suggestions from  #35996

### CODE REVIEW

File: doc/src/manual/mathematical-operations.md [hunk 1]
```diff
@@ -65,17 +59,20 @@ This is useful for preventing the propagation of `NaN` values in quantities that
...
-| `!x`       | not                                   |
+| `!x`       | negation                              |
 | `x && y`   | [short-circuiting and][short-circuit] |
 | `x \|\| y` | [short-circuiting or][short-circuit]  |
 
+Negation changes `true` to `false` and vice versa. The short-circuiting opeations are explained on the linked page.
+
 Note that `Bool` is an integer type and all the usual promotion rules and numeric operators are also defined on it.
 
 [short-circuit]: https://docs.julialang.org/en/v1/manual/control-flow/#Short-Circuit-Evaluation-1
```
Comment (ViralBShah) [inline doc/src/manual/mathematical-operations.md:74]: Could you also update this link to be an `@ref` rather than the URL?

**Comment by ViralBShah** [comment]
Not sure why all the testers are failing. Perhaps also rebase to master?

**Comment by cmcaine** [comment]
Rebased and updated the links :)

**Comment by ViralBShah** [comment]
Something in the PR is failing the tests.

**Comment by cmcaine** [comment]
The errors on tester_win{32,64} just seem to be curl timing out fetching stuff. Not plausibly anything to do with the PR.

tester_linuxaarch64 hadn't finished at the time I wrote this, but all other buildbots reported success.

**Pull request merged**